auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -140,6 +141,12 @@
         </div>
       </div>
 
+      <ExpandableHelp>
+        <template #content>
+          <CreateNewBusinessHelp />
+        </template>
+      </ExpandableHelp>
```

Comment:
I think this should go in the view header title block. This might fix the extra whitespace above it, which doesn't match the uxpin.

Right now:
![image](https://github.com/bcgov/sbc-auth/assets/10002889/a58aaf49-4d1c-4c5c-aa48-2413efd0f46f)

Uxpin:
![image](https://github.com/bcgov/sbc-auth/assets/10002889/9f06b028-34c1-4119-814c-3095f2a7aff4)